src/latexify/ast_utils_test.py

```diff
@@ -34,7 +34,6 @@ def test_make_attribute() -> None:
     )
 
 
-@test_utils.require_at_most(7)
```

Comment:
This test case (the whole function) is no longer meaningful, and should be completely removed.

Same as every other test case with a decoration `@test_utils.require_at_most(7)`

Reply:
I removed the legacy test case